.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,28 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python setup.py sdist bdist_wheel
+        twine upload dist/*
```

Comment:
No opinion from me on whether it's better or worse than using twine manually, but PyPA provides an action for PyPI deploys: https://github.com/pypa/gh-action-pypi-publish

Reply:
Could also consider adding another step for saving the dist files as job artifacts -- might help with debugging if something weird happens with the PyPI deploy.   Something like: 

```
      - uses: actions/upload-artifact@v2
        with:
          path: dist/*
```

Reply:
I noticed that pypi action as well. Ultimately I thought the explicit twine was just cleaner. I followed this example: https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-python#publishing-to-package-registries 

Good idea on the artifacts. I have added that.